Docs:Add procedure for viewing endpoint console (#4639)
EnMasseProject/enmasse: documentation/modules/ref-view-endpoint-info.adoc

// Module included in the following assemblies:
//
// assembly-using-console.adoc

[id='ref-view-endpoint-info-{context}']

= Viewing endpoint information using the {ConsoleName}

You can use the {ConsoleName} to view information about the endpoints configured for a given address space. You need this information to connect your messaging application to {ProductName}.

.Prerequisites
* You must be logged into the {ConsoleName}. For more information, see link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#logging-into-console-messaging[Accessing the {ConsoleName}].

ifdef::Asciidoctor[]
image::console-screenshot-endpoints.png[{ConsoleName}]
endif::Asciidoctor[]

ifndef::Asciidoctor[]
image::{imagesdir}/console-screenshot-endpoints.png[{ConsoleName}]
endif::Asciidoctor[]

.Messaging endpoint information reference table
[cols="50%a,50%a",options="header"]
|===
|Column |Description
|Name |Displays the name of the endpoint.
|Type |Displays the type of endpoint. Valid values include:

`cluster`:: An endpoint accessible on the {KubePlatform} cluster through a cluster IP address.

> `route`:: Creates a cluster endpoint available through the {KubePlatform} route.

Reviewer: surround `route` with the if openshift idiom so it is excluded from the Kubernetes docs.

> `route`:: Creates a cluster endpoint available through the {KubePlatform} route.

Reviewer: ```suggestion
`route`:: An {KubePlatform} route available outside the cluster.
```

> `loadbalancer`:: Creates a cluster endpoint available through the `LoadBalancer` service.

Reviewer: ```suggestion
`loadbalancer`:: A `LoadBalancer` service integrating with an external load balancer.
```

ifeval::["{cmdcli}" == "oc"]
For more information, see the following {KubePlatform} documentation:

> * link:https://docs.openshift.com/container-platform/4.4/networking/routes/route-configuration.html[Route Configuration]

Reviewer: Let's use vars for the urls.  That way it'll be less work when there is a version bump.
